Formula/g/go-swag.rb

```diff
@@ -0,0 +1,46 @@
+class GoSwag < Formula
```

Comment:
Why not `swag`? Repology recognizes it: https://repology.org/project/swag/versions

Reply:
I see, yeah, we should do swag then :)

I was looking at https://repology.org/project/go%3Aswag/versions